### DISCUSSION TITLE
Async DNS

### DISCUSSION THREAD

**Discussion by ghost** [micronaut-projects/micronaut-core#5369] [Q&A] [answered]
Hi folks,

Is the Micronaut HTTPClient using the Netty async DNS or the sync resolver?
And if it's using the sync apis, how can we control this to use the async ones?

Thanks!

**Comment by graemerocher** [comment, marked as answer]
Looks like `DefaultAddressResolverGroup` uses `DefaultNameResolver` by default. Would be worth an issue report with a feature request. Shouldn't be hard to add support for this.
